_inc/client/at-a-glance/scan.jsx

```diff
@@ -41,26 +43,47 @@ const DashScan = React.createClass( {
 			const threats = this.props.getScanThreats();
 			if ( threats !== 0 ) {
 				return(
-					<DashItem label="Security Scan (VaultPress)" status="is-error">
-						<h3>Uh oh, { threats } found!</h3>
-						<p className="jp-dash-item__description"><a href={ ctaLink }>Do something.</a></p>
+					<DashItem label={ labelName } status="is-error">
+						<h3>{
+							__(
+								'Uh oh, %(number)s threat found.', 'Uh oh, %(number)s threats found.',
+								{
+									count: threats,
+									args: {
+										number: threats
+									}
+								} )
+						}</h3>
+						<p className="jp-dash-item__description">
+							{ __( '{{a}}Go to VaultPress.com to do something.{{/a}}', { components: { a: <a href={ ctaLink } /> } } ) }
```

Comment:
Not specific to this PR, but this string definitely needs updating to something more useful :)